dali/kernels/CMakeLists.txt

```diff
@@ -19,15 +19,8 @@ add_subdirectory(common)
 add_subdirectory(imgproc)
 
 # Get all the source files and dump test files
-file(GLOB tmp *.cc *.cu)
-set(DALI_KERNEL_SRCS ${DALI_KERNEL_SRCS} ${tmp})
-file(GLOB tmp *_test.cc *_test.cu)
-remove(DALI_KERNEL_SRCS "${DALI_KERNEL_SRCS}" ${tmp})
-
-if (BUILD_TEST)
-  # get all the test srcs
-  file(GLOB tmp *_test.cc *_test.cu)
-endif()
+collect_headers()
```

Comment:
I think we should be explicitly passing the variable to fill

Reply:
Done